swarmrl/intrinsic_reward/rnd_configs.py

```diff
@@ -0,0 +1,138 @@
+"""
+Configurations for the Random Network Distillation (RND) intrinsic reward.
+
+Notes
+-----
+https://arxiv.org/abs/1810.12894
+"""
+
+from dataclasses import dataclass, field
+from typing import Optional
+
+import optax
+from flax import linen as nn
+from znnl.distance_metrics import DistanceMetric, OrderNDifference
+from znnl.loss_functions import MeanPowerLoss
+from znnl.training_strategies.loss_aware_reservoir import LossAwareReservoir
+from znnl.training_strategies.simple_training import SimpleTraining
+
+from swarmrl.networks.network import Network
+
+
+class RNDArchitecture(nn.Module):
```

Comment:
This can be changed of course. 
The RND architecture can be passed to the config for target and predictor model. 
Per default it is just set to the one implemented there. Also to provide one example for a user.

Reply:
added info in [83a10ce](https://github.com/SwarmRL/SwarmRL/pull/75/commits/83a10cefd2c3f2caf48f4d7c7535407716bd75c9)